.github/dependabot.yml

```
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  # We use pnpm for package management.
+  - package-ecosystem: 'npm'
+    # We only have a package.json in the root.
+    directory: '/'
+    # Check weekly to prevent too many PRs.
+    schedule:
+      interval: 'weekly'
+    allow:
+      # Starting slow by only checking our own packages for updates.
+      - dependency-name: '@metaplex-foundation/*'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)